[CSS] Hide `new-user-avatar-cta` protip box

zackschuster opened the pull request:
Hides the annoying 'Pro Tip! Updating your profile...' box on the personal profile page (appears whenever your name isn't set)

sindresorhus commented on extension/content.css:
```diff
@@ -514,3 +514,7 @@ hidden content area created to increase hover target
 	margin-top: 8px;
 }
+
+.new-user-avatar-cta {
```
Should have a comment description like other overrides.

zackschuster commented:
Fixed and squashed. Unfortunately, I'm at work right now and working around our bizarre proxy settings is proving too much, so I can't update to HEAD. I can do that when I get home.

zackschuster commented:
okay, i've updated to master. should be good to go now.

sindresorhus commented:
Look at the diff. You resolved the conflict incorrectly. It's now invalid CSS.

zackschuster commented:
whoops. did a pull from your master and didn't get a merge conflict, so i assumed it was fine. my apologies!
